modules/serverless-request-log-parameters.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/serverless-config-log-settings.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-request-log-parameters_{context}"]
+= Parameters of request logging
+
```

Comment:
This is not critical to do now and I will still merge if this isn't addressed by EOD today, but I think an introduction to the table/module + maybe a title for the table would be best here:

```suggestion

The following table describes parameters that can be used to configure request logging.

.Request logging parameters
```